fiftyone/multimodal/server/routes.py

```
@@ -0,0 +1,95 @@
+
+        return Response(
+            inventory.SerializeToString(),
+            media_type=PROTOBUF_MEDIA_TYPE,
+        )
+
+
+class PlaybackPlanEndpoint(HTTPEndpoint):
+    """Playback plan query endpoint."""
+
+    @decorators.route
+    async def post(self, request: Request, data: dict) -> Response:
```

Ruff (0.15.12)

[warning] Unused method argument: `request`

(ARG002)